### ISSUE TITLE
wikipathways and Reactome are swapped in the enrichment analysis

### ISSUE THREAD

**Issue by djinnome** [covid19kg/Analysis#2]
First of all, thank you so much for making this curated resource available.  I have been looking for exactly such a knowledge graph, and I would be happy to contribute to extending it so that it can capture viral life cycle, and additional host response pathways

Second of all, I noticed that the wikipathway identifiers looked like reactome ids, and vice versa. I believe the source of the error can be found in the jupyter cell: `Input[12]`

```python
REACTOME_URL = PREFIX + "wikipathways_geneset_final.gmt"
WIKIPATHWAYS_URL = PREFIX + "reactome_geneset_final.gmt"
```

A minor bug, and easily fixed.

**Comment by ddomingof** [comment]
Oups! Thanks for the catch. Will change it now. Feel free to add files to it and make a pull request, we are happy that people use it and contribute it.

**Comment by ddomingof** [comment]
Fix with b32ee3b9a1fa07866bbb41eae31fa130e892202a
